javascript/javascript3/week1/weather.js

```diff
@@ -0,0 +1,75 @@
+const cityButton = document.getElementById("city-name");
+
+cityButton.addEventListener("click", () => {
+  let url =
+    "http://api.openweathermap.org/data/2.5/weather?q=" +
+    document.querySelector("input").value +
```

Comment:
separate this out in variable and check for empty or null and then assign to URL